lib/evss/mhvcf/client.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+require 'evss/auth_headers'
+require 'evss/mhvcf/configuration'
+require 'evss/mhvcf/get_in_flight_forms_request_form'
+require 'evss/mhvcf/mhv_consent_form_request_form'
+
+module EVSS
+  module MHVCF
+    class Client < Common::Client::Base
+      configuration EVSS::MHVCF::Configuration
+
+      attr_accessor :auth_headers
+
+      def user(user)
```

Comment:
This method feels a bit strange to me, the requests require the auth headers so the caller always has to call this method first (as shown here https://github.com/department-of-veterans-affairs/vets-api/pull/620/files#diff-eea1c2cef1430765a5ec1145527453a6R15). Could this be moved to the initialization of the client or passed to the configuration somehow?